apps/web/src/components/views/CompanyView/index.tsx

```diff
@@ -1,50 +1,68 @@
-import { Button } from "@dotkomonline/ui"
 import { FC } from "react"
-
-import PortableText, { PortableTextProps } from "@/components/molecules/PortableText"
-
-import CompanyInterestProcess from "./CompanyInterestProcess"
-import OurProducts from "./OurProducts"
-import { Title } from "@dotkomonline/ui"
-
-export type Content = { content: PortableTextProps["blocks"] }
+import { Company, Event } from "@dotkomonline/types"
+import { Icon } from "@dotkomonline/ui"
+import { CompanyEventList } from "./CompanyEventList"
+import Image from "next/image"
 
 interface CompanyViewProps {
-  companyContent: Content[]
+  company: Company
+  events?: Event[]
+  isLoadingEvents: boolean
 }
+
 export const CompanyView: FC<CompanyViewProps> = (props: CompanyViewProps) => {
-  const [header, interest, product, info, contact] = props.companyContent
+  const { name, description, phone, email, website, location, type, image } = props.company
+
+  const icons = [
+    { icon: "material-symbols:location-on", text: location, href: null },
+    { icon: "ph:globe-bold", text: "Nettside", href: website },
+    { icon: "material-symbols:mail", text: email, href: `mailto:${email}` },
+    { icon: "material-symbols:phone-enabled", text: phone, href: `tel:${phone}` },
+  ]
 
   return (
-    <div className="flex flex-col gap-6 text-center">
-      <div className="bg-amber-2 w-full rounded-lg">
-        <div className="mx-auto flex h-[520px] max-w-[768px] flex-col items-center p-4 md:h-[300px] lg:h-[220px]">
-          <Title className="text-slate-1 mt-5 mb-4 text-4xl leading-[1.4]">
-            Er din bedrift på jakt etter skarpe IT-
-            <span style={{ backgroundPosition: "50% 88%" }} className="bg-[url(/for-company-text-decor.svg)]">
-              studenter?
-            </span>
-          </Title>
-          <PortableText className="prose text-center" blocks={header.content} />
+    <div className="mx-auto mb-20 flex w-[90vw] max-w-screen-lg flex-col gap-y-16">
+      <div className="flex flex-col gap-y-6">
+        <div className="border-blue-8 flex w-full items-end justify-between gap-x-2 border-b-2 pb-2">
+          <h1>{name}</h1>
+          <div className="text-blue-11 text-2xl">{type}</div>
+        </div>
+
+        <div className="grid gap-x-12 gap-y-6 sm:grid-cols-[18rem_minmax(100px,_1fr)] md:grid-cols-[24rem_minmax(100px,_1fr)]">
+          <div className="border-blue-7 flex h-fit flex-col gap-y-3 rounded-lg border-none sm:gap-y-2">
+            {image && (
+              <div className="relative mb-4 h-64 w-full overflow-hidden rounded-lg bg-[#fff]">
+                <a href={website} target="_blank" rel="noreferrer">
+                  <Image src={image} alt="Company logo" layout="fill" objectFit="contain" className="w-full" />
+                </a>
+              </div>
+            )}
+
+            <div className="text-blue-12 flex flex-col gap-y-2 px-1 text-lg [&>*]:flex [&>*]:items-center [&>*]:gap-x-2">
```

Comment:
Why not put items-center and gap-x-2 on the divs below?

Reply:
That was some old styling code from a bit earlier when the div contained multiple hardcoded divs. Fixed it now :)